src/main/java/com/paweldampc/service/CarsService.java

```diff
@@ -23,31 +25,31 @@ public CarsService(CarsRepository carsRepository, CarMapper carMapper) {
     }
 
     public List<Car> findAllToRepair() {
-        List<Car> allCarsToRepair = carsRepository.getCars().stream().filter(car -> car.getRepaired().equals(false)).collect(Collectors.toList());
+        List<Car> allCarsToRepair = carsRepository.getCars().stream()
+                .filter(car -> car.getRepaired().equals(false))
```

Comment:
metoda `getRepaired()` teoretycznie może zwrócić null, więc żeby przypadkiem nie dostać NullPointerException tutaj, to lepiej odwrócić equals:
```suggestion
                .filter(car -> Boolean.FALSE.equals(car.getRepaired()))
```
Podobnie w drugiej metodzie.